geode-core/src/main/java/org/apache/geode/management/internal/cli/result/model/TabularResultModel.java

```diff
@@ -43,6 +43,14 @@ public TabularResultModel accumulate(String column, String value) {
     return this;
   }
 
+  public void addMemberStatusResultRow(String member, String result, String message) {
```

Comment:
I am kind of concerned of moving this method here. The headers are so specific to each command though.